fair/tools/unifiedtools.py

```diff
@@ -1,6 +1,9 @@
 import numexpr as ne
 import numpy as np
 import pandas as pd
+import pyam as pyam
+
+from ..ancil.units import *
```

Comment:
```suggestion
from ..ancil.units import Units
```

Please don't ever use `*` imports, they're a nightmare

Reply:
Fair enough!